README.md

```diff
@@ -1 +1,325 @@
-# final-capstone-project-frontend
\ No newline at end of file
+
```

Comment:
_note:  I couldn't find an open PR on the backend project.  I'll put my comments here:_

## Backend comments

- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍



- [ ] If I run the tests according to your instructions, this happens:

![image](https://user-images.githubusercontent.com/57421823/206779092-fe060b23-1f93-47f4-a130-9cf577fd193e.png)

Since all your tests are rswag related, I suggest instructing your users to run them like this:

```
rake rswag
```

Result:

![image](https://user-images.githubusercontent.com/57421823/206779350-72591fc7-af16-426e-9dc6-2224d42f3e08.png)

That way, your users will never have a failed example. 👍 

- [ ] Please, add instructions to your backend readme about how to setup the database, which is needed for your users to run the project (e.g. rails db:create, rails db:migrate, rails db:seed).

- [ ] In your controllers, there is some old commented code.  Kindly remove it to improve the repos cleanliness.